### DISCUSSION TITLE
Analyzing chunk and identifying the responsible packages

### DISCUSSION THREAD

**Discussion by Jagadeeswarh** [vercel/next.js#28072] [Help] [answered]
We aim to increase the score for the light house, however the light house complains that [chunk](https://pastebin.pl/view/0c9a1d37) is not used. It's from the next js I think, aid is appreciated.

**Comment by jamesmosier** [comment, marked as answer]
[See this Discussion](https://github.com/vercel/next.js/discussions/13682#discussioncomment-21797) which describes what Lighthouse does but most importantly points out:

> It is somewhat inevitable that some subset of JavaScript will be unused when using someone's library

As far as your specific application, it is hard to say without seeing the code.
